packages/app/components/Queue/UpdateQuestionPopover.tsx

```diff
@@ -0,0 +1,23 @@
+import { Popconfirm } from "antd";
+import { ReactElement } from "react";
+
+interface UpdateQuestionPopoverProps {
+  onConfirm: () => void;
+  children: ReactElement;
+}
+
+export default function UpdateQuestionPopover({
```

Comment:
Not 100% sure this component is worth it. It's not really doing anything?

Reply:
matthias came into my brian and told me "you have common functionality," but I'm aight with removing it if anyone else has a strong opinion

Reply:
"into my brian"

Reply:
shhh i'm TRYING